zum-back-end/api/index.js

```diff
@@ -0,0 +1,157 @@
+const fs = require("fs");
+const crypto = require("crypto");
+const puppeteer = require("puppeteer");
+const express = require("express");
+const router = express.Router();
+
+/* 랭킹 데이터 */
+router.get("/best", (req, res, next) => {
+  try {
+    let data = fs.readFileSync("cache/json/best.json");
+    data = JSON.parse(data);
+    res.json(data);
+  } catch (err) {
+    res.status(500).send(`Throw Exception -> ${err}`);
+  }
+});
+
+/* 카테고리별 데이터 */
+router.get("/content/:category", (req, res, next) => {
+  try {
+    // Data Check for Security
+    switch (req.params.category) {
+      case "life":
+      case "food":
+      case "culture":
+      case "travel":
+        break;
+      default:
+        throw new Error("Violating security rules");
+    }
```

Comment:
```suggestion
    if (!['life', 'food', 'culture', 'tavel'].includes(req.params.category)) {
      throw new Error("Violating security rules");
    }
```
이렇게 줄일 수 있습니다.
 Error 로그도 특별한 경우가 아니라면 한글로 해주세요! 그게 에러가 발생했을 때 디버그 하기가 더 쉽답니다.